### ISSUE TITLE
`mix dialyzer` reports dependency missing

### ISSUE THREAD

**Issue by alappe** [jeremyjh/dialyxir#511]
### Environment

* Elixir & Erlang/OTP versions (elixir --version):
```
Erlang/OTP 26 [erts-14.0.2] [source] [64-bit] [smp:10:10] [ds:10:10:10] [async-threads:1] [jit]
Elixir 1.15.1 (compiled with Erlang/OTP 24)
```

* Which version of Dialyxir are you using? (cat mix.lock | grep dialyxir): `"dialyxir": {:hex, :dialyxir, "1.3.0", "fd1672f0922b7648ff9ce7b1b26fcf0ef56dda964a459892ad15f6b4410b5284", [:mix], [{:erlex, ">= 0.2.6", [hex: :erlex, repo: "hexpm", optional: false]}], "hexpm", "00b2a4bcd6aa8db9dcb0b38c1225b7277dca9bc370b6438715667071a304696f"},`

### Current behavior

`mix dialyzer` in this project prints

```
DEPENDENCY MISSING
------------------------
If you are reading this message, then Elixir and Erlang are installed but the
Erlang Dialyzer is not available. Probably this is because you installed Erlang
with your OS package manager and the Dialyzer package is separate.

On Debian/Ubuntu:

  `apt-get install erlang-dialyzer`

Fedora:

   `yum install erlang-dialyzer`

Arch and Homebrew include Dialyzer in their base erlang packages. Please report a Github
issue to add or correct distribution-specific information.
```

but using e.g. 

```
$ iex -S mix
Erlang/OTP 26 [erts-14.0.2] [source] [64-bit] [smp:10:10] [ds:10:10:10] [async-threads:1] [jit]

Interactive Elixir (1.15.1) - press Ctrl+C to exit (type h() ENTER for help)
iex(1)> Code.ensure_loaded?(:dialyzer)
true
iex(2)>
```

The elixir/erlang is installed on macOS via asdf, `.tool-versions`:

```
elixir 1.15.1
erlang 26.0.2
```

It's an umbrella project and I run it in the umbrella itself.

### Expected behavior

After I upgraded dialyxir, elixir and erlang I expected it to work the same as before.

**Comment by alappe** [comment]
Alright, thank you for the feedback.

**Comment by jeremyjh** [comment]
Fix published in 1.4: https://hex.pm/packages/dialyxir/1.4.0
